src/sentry/utils/snuba.py

```diff
@@ -1053,6 +1053,9 @@ def _bulk_snuba_query(
             for k, v in quota_allowance_summary["rejected_by"].items():
                 span.set_tag(k, v)
 
+            if "rejected_by" in quota_allowance_summary and quota_allowance_summary["rejected_by"]:
+                logger.warning("Query is throttled")
```

Comment:
You probably want more info here. This is going to create a sentry issue. Think about the product developer looking at it, what are they supposed to do with this information?

Reply:
I think only rejected queries will create a sentry issue. The response.status of throttled queries should still be 200. Since there is no sentry issue that displays all the helpful metadata for throttled queries, I figured it doesn't hurt to just log the entire response.data, because we don't know what info product developers can need.

Reply:
logging.warning is how you create warnings in sentry (which become sentry issues)